diesel_derives/src/attr.rs

```diff
@@ -5,8 +5,8 @@ use util::*;
 
 #[derive(Debug)]
 pub struct Attr {
-    column_name: Option<syn::Ident>,
-    field_name: Option<syn::Ident>,
+    pub column_name: Option<syn::Ident>,
+    pub field_name: Option<syn::Ident>,
```

Comment:
Why do we need to make this public? Things should be using the `field_name()` method.

Reply:
Same answer...